processor/src/main/java/org/mapstruct/ap/internal/util/Message.java

```diff
@@ -89,6 +89,7 @@
     GENERAL_VALID_DATE( "Given date format \"%s\" is valid.", Diagnostic.Kind.NOTE ),
     GENERAL_INVALID_DATE( "Given date format \"%s\" is invalid. Message: \"%s\"." ),
     GENERAL_NOT_ALL_FORGED_CREATED( "Internal Error in creation of Forged Methods, it was expected all Forged Methods to finished with creation, but %s did not" ),
+    GENERAL_NO_SUITABLE_CONSTRUCTOR( "%s does not have a suitable empty constructor." ),
```

Comment:
I think we can omit the “suitable”, as you already mention “empty”. Maybe add “visible” or “accessible”?

Reply:
I agree "accessible" sounds better than "suitable". I'll change it. Do you think that we need to give a suggestion what they should do? To me it is pretty clear, but I don't know 😄

Reply:
Yes, I think it’s pretty obvious... 🙂